addons/interaction/CfgVehicles.hpp

```diff
@@ -368,6 +368,40 @@ class CfgVehicles {
         };
     };
 
+    class Car_F: Car{};
+    class Quadbike_01_base_F: Car_F {
+        class ACE_Actions: ACE_Actions {
+            class ACE_MainActions: ACE_MainActions {
+                class GVAR(flip) {
+                    displayName = CSTRING(Flip);
+                    condition = QUOTE(1 < (vectorUp _target) vectorDistance (surfaceNormal getPosATL _target) && {0 == {alive _x} count crew _target});
+                    statement = QUOTE([ARR_3(QQGVAR(flip),_target,_target)] call CBA_fnc_targetEvent);
+                };
+                class GVAR(push) {
+                    displayName = CSTRING(Push);
+                    condition = QUOTE(_target call FUNC(canPush));
+                    statement = QUOTE(call FUNC(push));
+                };
+            };
+        };
+    };
+    class Kart_01_Base_F: Car_F {
+        class ACE_Actions: ACE_Actions {
+            class ACE_MainActions: ACE_MainActions {
+                class GVAR(flip) {
+                    displayName = CSTRING(Flip);
+                    condition = QUOTE(1 < (vectorUp _target) vectorDistance (surfaceNormal getPosATL _target) && {0 == {alive _x} count crew _target});
```

Comment:
First condition should be made into a common function, as it repeats.

Reply:
I didn't want to move such a small piece of code to function.